src/components/action-buttons/action-buttons.component.tsx

```diff
@@ -60,22 +59,21 @@ function ActionButtons({ schema, t }: ActionButtonsProps) {
     try {
       await unpublishForm(form.uuid);
 
-      showToast({
+      showSnackbar({
         title: t('formUnpublished', 'Form unpublished'),
         kind: 'success',
-        critical: true,
-        description: `${form.name} ` + t('formUnpublishedSuccessfully', 'form was unpublished successfully'),
+        isLowContrast: true,
```

Comment:
Nit: In JSX, if you have a prop that receives a boolean value, you can omit the value if it is true. For example, the following snippets are functionally equivalent: 

```tsx
<Component isLowContrast />

<Component isLowContrast={true} />
```